tox_poetry_installer/logger.py

```diff
@@ -4,26 +4,26 @@
 gets really old fast, but more importantly it also makes the flow of the main code
 more difficult to follow because of the added complexity.
 """
-import tox
+import logging
 
 from tox_poetry_installer import constants
 
 
 def error(message: str):
     """Wrapper around :func:`tox.reporter.error`"""
```

Comment:
Can you update the docstrings in this module to reflect the change? Something like:
```python
"""Wrapper around :func:`logging.error` that prefixes the reporter prefix onto the message"""
```

Reply:
I updated the docstrings of all functions, with only the error wrapper mentioning the report prefix. Didn't like the idea of repeating the same text multiple times in the same file :sweat_smile: